rosapi/src/rosapi/objectutils.py

```diff
@@ -74,8 +85,13 @@ def get_typedef(type):
         return _get_special_typedef(type)
 
     # Fetch an instance and return its typedef
-    instance = ros_loader.get_message_instance(type)
-    return _get_typedef(instance)
+    try:
+        instance = ros_loader.get_message_instance(type)
+        type_def = _get_typedef(instance)
+        return type_def
+    except Exception as e:
```

Comment:
Let's only catch what can be thrown?

```suggestion
    except (InvalidModuleException, InvalidClassException) as e: